package.json

```diff
@@ -32,39 +32,39 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand"
   },
   "devDependencies": {
-    "@nestjs/common": "^10.3.3",
-    "@nestjs/platform-express": "^10.3.3",
-    "@nestjs/testing": "^10.3.3",
+    "@nestjs/common": "^10.3.8",
+    "@nestjs/platform-express": "^10.3.8",
+    "@nestjs/testing": "^10.3.8",
     "@types/express": "^4.17.21",
     "@types/jest": "^29.5.12",
-    "@types/lodash": "^4.14.202",
-    "@types/node": "^20.11.20",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
+    "@types/lodash": "^4.17.0",
+    "@types/node": "^20.12.7",
+    "@typescript-eslint/eslint-plugin": "^7.4.0",
+    "@typescript-eslint/parser": "^7.4.0",
     "dotenv": "^16.4.5",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.1.3",
-    "fastify": "^4.26.1",
+    "fastify": "^4.26.2",
     "jest": "^29.7.0",
-    "pg": "^8.11.3",
+    "pg": "^8.11.5",
     "prettier": "^3.0.3",
     "reflect-metadata": "^0.1.14",
     "rxjs": "^7.8.1",
     "sqlite3": "^5.1.7",
     "ts-jest": "^29.1.2",
     "ts-node": "^10.9.2",
     "typeorm": "^0.3.17",
-    "typescript": "^5.3.3"
+    "typescript": "^5.4.5"
   },
   "dependencies": {
     "lodash": "^4.17.21"
   },
   "peerDependencies": {
```

Comment:
I don't think we need to update peer deps at all to be honest 

it should be @nestjs/common ^10.0.0, etc... for others so we should support any major version in general

Reply:
Agreed, I will revert these changes, and then we can agree on a dependency versioning policy in general. We should test every combination of major versions of each package that we support in CI.

Reply:
I think at least do not update peers for now

other then that looks ok for me

Reply:
Ah sorry, I didn't do this! :D This is actually because I merged `main` into this branch. As you can see, this PR targets the `ppetzold:json-api` branch, which is behind `main`. So this will merely synchronize `ppetzold:json-api` with the changes already merged in `main`, by @ppetzold :)

Reply:
looks ok to me then ... maybe can be addressed later